### PR TITLE
feat(analysisType): Refactor 2

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b refactor-analysistypes-2
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
### CG and Tb, 👍 

```
[INFO] 2024/11/27 09:43:36 MIP_ANALYSE - To check status of job, please run 'squeue -j 7755950'
[INFO] 2024/11/27 09:43:36 MIP_ANALYSE - To cancel job, please run 'scancel 7755950'
[INFO] 2024/11/27 09:43:36 MIP_ANALYSE - Wrote: /home/proj/stage/rare-disease/cases/justhusky/analysis/justhusky_qc_sample_info.yaml
[INFO] 2024/11/27 09:43:36 MIP_ANALYSE - Wrote: /home/proj/stage/rare-disease/cases/justhusky/analysis/slurm_job_ids.yaml
[INFO] 2024/11/27 09:43:36 MIP_ANALYSE - Wrote: /home/proj/stage/rare-disease/cases/justhusky/analysis/justhusky_deliverables.yaml
Action running set for case justhusky
mip-dna run started!
```

Screenshot 2024-11-27 at 09 45 18

File: cg/constants/tb.py [hunk 1]
```diff
@@ -13,8 +15,8 @@ class AnalysisStatus:
 
 
 class AnalysisTypes(StrEnum):
-    WGS: str = "wgs"
-    WES: str = "wes"
-    TGS: str = "tgs"
-    WTS: str = "wts"
     OTHER: str = "other"
+    TGS: str = "tgs"
```
Comment (islean) [inline cg/constants/tb.py:19]:
```suggestion
    TGS: str = SeqLibraryPrepCategory.TARGETED_GENOME_SEQUENCING
```

**Review by islean — APPROVED** [review]
Looks good!

**Comment by henrikstranneheim** [comment]
Deployed to prod:
```
Log deploy... done.
cg, version 64.5.16

**Pull request merged**